Bottom: 891a16cb25ad39db51200ed1eb3262b8f982b9bd
Top:    0811012347b0773da571fc97d17714e8e0e598f4
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-11 10:41:02 -0700

rstream: Clarify use of async test option

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/rstream.c b/examples/rstream.c
index 825c4de..ecaa779 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -556,7 +556,7 @@ int main(int argc, char **argv)
 			printf("\t[-p port_number]\n");
 			printf("\t[-T test_option]\n");
 			printf("\t    s|sockets - use standard tcp/ip sockets\n");
-			printf("\t    a|async - asynchronous operation\n");
+			printf("\t    a|async - asynchronous operation (use poll)\n");
 			printf("\t    b|blocking - use blocking calls\n");
 			printf("\t    n|nonblocking - use nonblocking calls\n");
 			printf("\t    v|verify - verify data\n");
